sdk/highlight-ruby/highlight/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
 ## 0.1.2
 
 - Add ability to set `service_name` and `service_version`
+
+## 0.1.4
+
+
+## 3.3.2
```

Comment:
Just checking this was meant to be 3.3.2, since saw we are updating to 0.1.4 below?

Reply:
good catch!